### PR TITLE
Update the workflow

### PR DESCRIPTION
**Pull request by ThomasHMAC**
Hey @josephmje, 

Everything is resolved.

### CODE REVIEW

File: tractify/utils.py [hunk 1]
```diff
@@ -193,7 +193,8 @@ def collect_dmripreproc_output(dmriprep_dir, subject_id, session_id = None):
     dmri_output['eddy_file'] = os.path.join(dmri_files_path, 'eddy_corrected.nii.gz')
     dmri_output['bvec'] = os.path.join(dmri_files_path, 'eddy_corrected.eddy_rotated_bvecs')
     dmri_output['bval'] = os.path.join(dmri_files_path, next(f for f in dmri_files if f.endswith('.bval')))
-    dmri_output['dwi_mask'] = os.path.join(dmri_files_path, next(f for f in dmri_files if f.endswith('brain_mask.nii.gz')))
+    #dmri_output['eddy_avg_b0'] = os.path.join(dmri_files_path, next(f for f in dmri_files if f.endswith('eddy_corrected_avg_b0.nii.gz')))
```
Comment (josephmje) [inline tractify/utils.py]: can you remove this commented out line

File: tractify/workflows/base.py [hunk 2]
```diff
@@ -72,7 +72,8 @@ def init_single_subject_wf(subject_id, name, parameters):
             inputspec.inputs.output_dir = parameters.output_dir
             inputspec.inputs.t1_file = t1_file
             inputspec.inputs.eddy_file = dmripreproc_output['eddy_file']
-            inputspec.inputs.dwi_mask = dmripreproc_output['dwi_mask']
+            inputspec.inputs.eddy_avg_b0 = dmripreproc_output['eddy_avg_b0']
+            #inputspec.inputs.dwi_mask = dmripreproc_output['dwi_mask']
```
Comment (josephmje) [inline tractify/workflows/base.py]: remove commented out line

File: tractify/workflows/tractography/base.py [hunk 3]
```diff
@@ -30,7 +30,8 @@ def init_tract_wf():
                 "t1_file",
                 "eddy_file",
                 "eddy_mask",
-                "dwi_mask",
+                #"dwi_mask",
```
Comment (josephmje) [inline tractify/workflows/tractography/base.py]: remove commented out line

File: tractify/workflows/tractography/base.py [hunk 4]
```diff
@@ -129,7 +130,8 @@ def gen_tuple(item1, item2):
                 inputnode,
                 flirt,
                 [
-                    ("dwi_mask", "reference")
+                    #("dwi_mask", "reference") ## this was not the right input_file for flirt
```
Comment (josephmje) [inline tractify/workflows/tractography/base.py]: remove commented out line

**Review by josephmje — CHANGES_REQUESTED** [review]

File: tractify/utils.py (hunk 1 above)
Comment (ThomasHMAC) [reply tractify/utils.py]: sure, i also removed the rest of the extra comments

**Review by josephmje — APPROVED** [review]

**Comment by josephmje** [comment]
Great work @ThomasHMAC !

**Pull request merged**